docs/pages/enterprise/workflow/ssh-approval-jira-cloud.mdx

```diff
@@ -82,19 +82,34 @@ In the webhook settings page, make sure that the webhook will only send Issue Up
 
 We recommend installing Teleport plugins alongside the Teleport Proxy. This is an ideal
 location as plugins have a low memory footprint, and will require both public internet access
-and Teleport Auth access. We currently only provide linux-amd64 binaries, you can also
-compile these plugins from [source](https://github.com/gravitational/teleport-plugins/tree/master/access/jira).
+and Teleport Auth Service access. 
 
-```code
-$ wget https://get.gravitational.com/teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-$ tar -xzf teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
-$ cd teleport-access-jira/
-$ ./install
-$ which teleport-jira
-# /usr/local/bin/teleport-jira
-```
-
-Run `./install` in from 'teleport-jira' or place the executable in the appropriate `/usr/bin` or `/usr/local/bin` on the server installation.
+<Tabs>
+<TabItem label="Download">
+  ```code
+  $ curl -L -O https://get.gravitational.com/teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ tar -xzf teleport-access-jira-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ cd teleport-access-jira
+  $ ./install
+  ```
+</TabItem>
+<TabItem label="From Source">
+  To install from source you need `git` and `go` installed. If you do not have Go installed, visit the Go [downloads page](https://go.dev/dl/).
+
+  ```code
+  # Checkout teleport-plugins
+  $ git clone https://github.com/gravitational/teleport-plugins.git
+  $ cd teleport-plugins/access/jira
+  $ make
+  ```
+Run `./install` from `teleport-jira` or place the executable in `/usr/bin` or `/usr/local/bin` on the server installation.
+</TabItem>
+<TabItem label="Docker">
+  ```code
+  $ docker pull quay.io/gravitational/teleport-plugin-jira:(=teleport.plugin.version=)
```

Comment:
Is there supposed to be an image for `quay.io/gravitational/teleport-plugin-jira:9.0.4` ? The `docker pull` command fails and I don't see any images/tags in https://quay.io/repository/gravitational/teleport-plugin-jira.

Reply:
The repositories were renamed, I forgot to mirror them externally. They should exist now.